skins/modal/index.ess

```diff
@@ -0,0 +1,18 @@
+.Modal
```

Comment:
let's remove this selector and dedent the whole thing. that way the developer has control over the selector name